Makefile

```diff
@@ -410,34 +430,39 @@ ifneq ($(RESTY_IMAGE_BASE),src)
 endif
 
 build-test-container:
+ifneq ($(RESTY_IMAGE_BASE),src)
 	touch test/kong_license.private
 	ARCHITECTURE=amd64 \
+	KONG_PACKAGE_NAME=$(KONG_PACKAGE_NAME) \
 	PACKAGE_TYPE=$(PACKAGE_TYPE) \
 	RESTY_IMAGE_BASE=$(RESTY_IMAGE_BASE) \
 	RESTY_IMAGE_TAG=$(RESTY_IMAGE_TAG) \
 	KONG_VERSION=$(KONG_VERSION) \
-	KONG_PACKAGE_NAME=$(KONG_PACKAGE_NAME) \
-	KONG_TEST_IMAGE_NAME=$(KONG_TEST_IMAGE_NAME) \
+	DOCKER_RELEASE_REPOSITORY=$(DOCKER_RELEASE_REPOSITORY) \
+	KONG_TEST_CONTAINER_TAG=$(KONG_TEST_CONTAINER_TAG) \
 	DOCKER_KONG_VERSION=$(DOCKER_KONG_VERSION) \
 	DOCKER_BUILD_PROGRESS=$(DOCKER_BUILD_PROGRESS) \
 	DOCKER_LABELS="$(DOCKER_LABELS)" \
 	test/build_container.sh
+	docker tag $(DOCKER_RELEASE_REPOSITORY):amd64-$(KONG_TEST_CONTAINER_TAG) \
+		$(DOCKER_RELEASE_REPOSITORY):$(KONG_TEST_CONTAINER_TAG)
+	docker tag $(DOCKER_RELEASE_REPOSITORY):amd64-$(KONG_TEST_CONTAINER_TAG) \
+		$(KONG_TEST_IMAGE_NAME)
 ifeq ($(BUILDX),true)
-	DOCKER_MACHINE_NAME=$(shell docker-machine env $(DOCKER_MACHINE_ARM64_NAME) | grep 'DOCKER_MACHINE_NAME=".*"' | cut -d\" -f2) \
-	DOCKER_TLS_VERIFY=1 \
-	DOCKER_HOST=$(shell docker-machine env $(DOCKER_MACHINE_ARM64_NAME) | grep 'DOCKER_HOST=".*"' | cut -d\" -f2) \
-	DOCKER_CERT_PATH=$(shell docker-machine env $(DOCKER_MACHINE_ARM64_NAME) | grep 'DOCKER_CERT_PATH=".*"' | cut -d\" -f2) \
+	docker run --rm --privileged multiarch/qemu-user-static --reset -p yes && \
```

Comment:
So, does this PR do away with docker-machine? If so, we should also strip out the docker-machine install.

Reply:
It does not.

We use docker-machine / a remote arm worker for the purposes of building Kong arm64. The final step of packaing and placing said package onto the host system `output/` directory is via qemu.